src/tools/cursors/MouseCursor.js

```diff
@@ -0,0 +1,77 @@
+import toolColors from '../../stateManagement/toolColors.js';
+import { modules } from '../../store/index.js';
+
+const cursorModule = modules.cursor;
+
+export default class MouseCursor {
+  constructor(iconGroupString, options) {
+    this.iconGroupString = iconGroupString;
+    this.options = Object.assign(
+      {},
+      cursorModule.getters.defaultOptions(),
+      options
+    );
+  }
+
+  get iconSVG() {
+    const svgString = this._generateIconSVGString();
+
+    return new Blob([svgString], { type: 'image/svg+xml' });
+  }
+
+  get iconWithPointerSVG() {
+    const svgString = this._generateIconWithPointerSVGString();
+
+    return new Blob([svgString], { type: 'image/svg+xml' });
+  }
+
+  get iconSVGString() {
+    return this._generateIconSVGString();
+  }
+
+  get iconWithPointerString() {
+    return this._generateIconWithPointerSVGString();
```

Comment:
Can we update the string methods to take optional `options` object (default's to internal values)? That way we can grab an icon without updating the properties used by an active cursor.

IE. I want 3 or 4 display representation w/ different values, but the SVG Cursor on the canvas should continue to look the same.

Reply:
I like this, sure.

Reply:
Done.